src/main/groovy/io/saagie/plugin/dataops/utils/SaagieUtils.groovy

```diff
@@ -407,7 +403,7 @@ class SaagieUtils {
             }
         ''', gqVariablesWithNullFile )
 		
-		return buildMultipartRequestFromQuery( createProjectJob )
+		return buildMultipartRequestFromQuery( createProjectJob, job, jobVersion )
```

Comment:
please use the same spacing in the whole project (remove the space between the parenthesis)
`(createProjectJob, job, jobVersion)`